lib/generators/oneshot/oneshot_generator/configuration.rb

```diff
@@ -1,10 +1,6 @@
 class OneshotGenerator < Rails::Generators::NamedBase
   class Configuration
-    attr_accessor :body
-
-    def initialize
-      @body = nil
-    end
```

Comment:
This assignment is unnecessary when it uses `attr_accessor`.
It completely works well, with no warning.